app/lib/app/SecvisogramPage.js

```diff
@@ -74,6 +74,12 @@ const SecvisogramPage = () => {
     }
   }, [t])
 
+  const [backendVersion, setBackendVersion] = React.useState('')
+
+  React.useEffect(() => {
+    backend.getAboutInfo().then((info) => setBackendVersion(info.version))
+  })
```

Comment:
I wonder if we could somehow reduce the times this request is sent. @domachine maybe you've got a nice idea?

Reply:
Since the dependency array for `useEffect()` is missing it will trigger on every render. As far as I can see there are no dependencies, therefore an empty one should do the job:

```javascript
React.useEffect(() => {
  // ...
}, [])
```